ci-scripts/yocto2.groovy

```diff
@@ -131,10 +131,24 @@ void runYoctoCheckLayer(String yoctoDir) {
 
 
 void archiveCache(String yoctoDir, boolean doArchiveCache, String yoctoCacheArchivePath) {
-    if (doArchiveCache && yoctoCacheArchivePath?.trim()) {
-        stage("Archive cache") {
-            sh "rsync -trpgO ${yoctoDir}/build/downloads/ ${yoctoCacheArchivePath}/downloads/"
-            sh "rsync -trpgO ${yoctoDir}/build/sstate-cache/ ${yoctoCacheArchivePath}/sstate-cache"
+    stage("Verify NFS") {
```

Comment:
I do not think this warrants a separate stage. Just have it be part of the "Archive cache" stage. We are not interested in measuring time it takes nor will we ever have it be anything other than "green".